lib/documents.js

```diff
@@ -2248,7 +2249,7 @@ Documents.prototype.writeAll = function writeAllDocuments(options) {
       this, options
   );
 };
-function writeAllDocumentsImpl(options) {
+function writeAllDocumentsImpl(options, done) {
```

Comment:
done is a test function, not a production application function

The test will need to configure the options with job success and failure callbacks.

In those callbacks, the test will call done().